Lets Do This/Views/Campaign/LDTCampaignListCampaignCell.m

```diff
@@ -23,6 +23,11 @@ @interface LDTCampaignListCampaignCell()
 @property (weak, nonatomic) IBOutlet NSLayoutConstraint *imageViewBottom;
 @property (weak, nonatomic) IBOutlet NSLayoutConstraint *imageViewTop;
 @property (weak, nonatomic) IBOutlet NSLayoutConstraint *titleLabelTopLayoutConstraint;
+@property (weak, nonatomic) IBOutlet NSLayoutConstraint *imageViewHeight;
+@property (weak, nonatomic) IBOutlet NSLayoutConstraint *taglineTop;
+@property (weak, nonatomic) IBOutlet NSLayoutConstraint *actionButtonHeight;
+@property (weak, nonatomic) IBOutlet NSLayoutConstraint *actionButtonTop;
+@property (weak, nonatomic) IBOutlet NSLayoutConstraint *actionButtonBottom;
```

Comment:
Added all these constraints in attempt to get self sizing cells for both expanded and collapsed states.

If we didn't have to worry about supporting iOS8, sounds like the `UIStackView` in iOS9 is the way to avoid this, and all of the new code in the `-(void)setExpanded:(BOOL)expanded`.